engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/content-host-errata.controller.js

```diff
@@ -135,8 +135,30 @@ angular.module('Bastion.content-hosts').controller('ContentHostErrataController'
 
         $scope.performViaRemoteExecution = function(customize) {
             var errataIds = $scope.selectedErrataIds();
-            $scope.errataActionFormValues.errata = errataIds.included.ids.join(',');
+            var sourceHtml = '';
+
+            $scope.errataActionFormValues.bulkErrataIncludedIds = errataIds.included.ids;
             $scope.errataActionFormValues.hostIds = $scope.host.id;
+
+            $.each(errataIds.included.ids, function(index, errataId) {
+                sourceHtml += '<input type="hidden" name="bulk_errata_ids[included][ids][]" value="' +
+                  errataId + '" />';
+            });
+
+            $('#errataActionForm').append(sourceHtml);
+
+            if ($scope.nutupane.table.allResultsSelected) {
+                sourceHtml = '<input type="hidden" name="bulk_errata_ids[included][search]" value="*" />';
```

Comment:
why are you passing * in here?     Shouldn't it be the value from the search box?